### PR TITLE
File Format for DTEB (CDS Entity Buffer)

### PR DESCRIPTION
**Pull request by BeckerWdf**

### CODE REVIEW

**Comment by schneidermic0** [comment]
Just to be sure: Does DTEB support all ABAP language versions including "keyUser"?

**Comment by BeckerWdf** [comment]
> Just to be sure: Does DTEB support all ABAP language versions including "keyUser"?

Good point. I will check.

**Comment by BeckerWdf** [comment]
> Just to be sure: Does DTEB support all ABAP language versions including "keyUser"?

It does only support "Standard ABAP" and "ABAP for Cloud Development". I fixed this.

File: file-formats/dteb/type/zif_aff_dteb_v1.intf.abap [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  TYPES:
+    "! <p class="shorttext">CDS Entity Buffer</p>
+    "! CDS entity buffer
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60,
```
[the remark this answers is not in the document]
Comment (BeckerWdf) [reply file-formats/dteb/type/zif_aff_dteb_v1.intf.abap:13]: Oh yes. You are right.

**Review by schneidermic0 — APPROVED** [review]
Looks good to me

**Pull request merged**